app/controllers/concerns/deploy_groups_common.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+module DeployGroupsCommon
+  extend ActiveSupport::Concern
```

Comment:
take out anything we don't need :)

On Tue, Aug 16, 2016 at 2:00 PM, Adam Panzer <notifications@github.com>
wrote:

> In app/controllers/concerns/deploy_groups_common.rb
> <https://github.com/zendesk/samson/pull/1198#discussion_r75019220>:
>
> > @@ -0,0 +1,8 @@
> > +# frozen_string_literal: true
> > +module DeployGroupsCommon
> > +  extend ActiveSupport::Concern
>
> in this case, no, but I might need it later?
>
> Take it out?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1198/files/d43bdda94cc05d08fa093f7c694b09953335fe89#r75019220>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZyX6PtX8MiIzN6Du_8L8KIb0Mxd-ks5qgiT9gaJpZM4Jhtog>
> .
>